src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php

```diff
@@ -60,18 +60,30 @@ public function GetHTML()
             }
         } else {
             $sOnClickEvent = $this->getOnClickEvent();
-            $html .= '<input type="hidden" id="'.TGlobal::OutHTML($this->name).'" name="'.TGlobal::OutHTML($this->name).'" value="'.TGlobal::OutHTML($sPropertyTableName).'" />';
-            $html .= '
-        <div style="border-bottom: 1px solid #A9C4E7;">
-          <div class="listBoxTopTitleRight"><img src="'.URL_CMS.'/images/boxTitleBgRight.gif" alt="" width="4" height="22" border="0" hspace="0" vspace="0" /></div>
-          <div class="listBoxTopTitleLeft"><img src="'.URL_CMS.'/images/boxTitleBgLeft.gif" alt="" width="4" height="22" border="0" hspace="0" vspace="0" /></div>
-          <div class="listBoxTop" data-fieldstate="'.$stateContainer->getState($this->sTableName, $this->name)."\" onClick=\"setTableEditorListFieldState(this, '{$sStateURL}');{$sOnClickEvent}\"><img src=\"".URL_CMS.'/images/icon_show_hide_list.gif" border="0" hspace="0" vspace="0" align="left" style="padding-top: 3px; padding-right: 8px; padding-left: 4px; ">'.TGlobal::Translate('chameleon_system_core.field_property.open_or_close_list').'</div>
-          <div class="cleardiv">&nbsp;</div>
-        </div>
-        <div id="'.TGlobal::OutHTML($this->name).'_iframe_block" class="listBoxBorder"><iframe id="'.TGlobal::OutHTML($this->name).'_iframe" width="100%" height="470" frameborder="0" style="display: none;"></iframe></div>';
+
+            $sEscapedName = TGlobal::OutHTML($this->name);
+
+            $html .= '<input type="hidden" id="' . TGlobal::OutHTML($this->name) . '" name="' . TGlobal::OutHTML($this->name) . '" value="' . TGlobal::OutHTML($sPropertyTableName) . '" />';
```

Comment:
Concatenation dots aren't surrounded by space - see https://symfony.com/doc/current/contributing/code/standards.html

Could you run PHP-CS-Fixer again on changed code?

Reply:
done